tests/coalaCITest.py

```diff
@@ -33,9 +33,7 @@ def test_nonexistent(self, debug=False):
         retval, stdout, stderr = execute_coala(
             coala.main, 'coala', '--non-interactive', '-c', 'nonex', 'test')
```

Comment:
arent they now? also i think its easier to review if u do it like that so pls do it like that :(